docs/routers/fritzbox.md

```diff
@@ -85,3 +84,90 @@ The following settings must be made:
     * **Local domain name (optional):** Fritz!Box uses **fritz.box**
 
 ![Screenshot der Conditional Forwarding Einstellungen](../images/conditional-forwarding.png)
+
+## Distribute Pi-hole as DNS server via IPv6
+
+Using this configuration, all clients will get Pi-hole's IPv6 offered as DNS server when connected to your Fritz!Box.
+
+### Stable IPv6 address for your Pi-hole
+
+By default most devices use the privacy extensions for IPv6 (RFC 4941). Those clients generate a random interface identifier while connecting to a network and rotate it after a set time delay. This is done to hide the clients MAC address on the internet.
+Therefore it is necessary to pick a stable IPv6 which does not change on its own.
+
+Most OS assign a stable interface identifiers for management purposes to an interface.
+There exist multiple standards and implementations how it is generated and how stable it actually is.
+If you are unsure, consult the documentation of your OS about the configuration.
+The following section will help you picking a suitable IPv6 address of your Pi-hole.
+To show all IPv6 addresses currently in use by your main network interface, open a terminal, substitute `eth0` with your main interface name (or omit) and execute the command
+
+```bash
+ip -6 address show eth0
+```
+
+The output should look like this but with different addresses
+
+```bash
+$ ip -6 address show eth0
+2: eth0: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 1500 qdisc fq_codel state UP group default qlen 1000
+    inet6 fd6f:95dc:3a80:e9b1:503e:b05a:21dc:cf0c/64 scope global temporary dynamic
+       valid_lft 7159sec preferred_lft 3559sec
+    inet6 fd6f:95dc:3a80:e9b1:4bc3:7bff:fe67:c175/64 scope global dynamic mngtmpaddr noprefixroute
+       valid_lft 7159sec preferred_lft 3559sec
+    inet6 2001:db8::1c5e:22fe:490c:1c31/64 scope global temporary dynamic
+       valid_lft 7159sec preferred_lft 3559sec
+    inet6 2001:db8::4bc3:7bff:fe67:c175/64 scope global dynamic mngtmpaddr noprefixroute
+       valid_lft 7159sec preferred_lft 3559sec
+    inet6 fe80::4bc3:7bff:fe67:c175/64 scope link noprefixroute
+       valid_lft forever preferred_lft forever
+```
+
+When picking an IPv6 address from that list:
+
+* avoid global unicast addresses (GUA) (range 2000::/3)  
+  Your ISP controls your GUA IPv6 prefix it, so it may change, either regularly or on router restarts.
+  In this example, don't use the third and fourth address, starting with 2001:
+* avoid Privacy Extension addresses (marked with temporary)  
+  The interface identifier portion of an IPv6 address is designed to change regularly, on some systems as often as every hour.
+  In this example, avoid the first and the third address.
+* prefer unique local addresses (ULA) (range `fd00::/8`) over the link-local address (range `fe80::/10`)  
+  You can control the ULA prefix and it is static. The later is only valid on a link and cannot be routed.
```

Comment:
"The later" may refer to "and it is static" in the preceeding sentence.
For clarity:
"A link-local address is only valid on a link and cannot be routed."